src/components/icon/index.tsx

```diff
@@ -27,6 +27,16 @@ export type IconProps = ImageProps &
      * whether the icon should flip horizontally on RTL
      */
     supportRTL?: boolean;
+    /**
+     * Width and height of the icon will be used only in case that the icon is svg image.
```

Comment:
This will only be shown in the width docs, maybe write something specific for each one?